components/form/Form.razor.cs

```diff
@@ -166,6 +179,52 @@ private async Task OnInvalidSubmit(EditContext editContext)
             await OnFinishFailed.InvokeAsync(editContext);
         }
 
+        private void OnFieldChanged(object sender, FieldChangedEventArgs args)
```

Comment:
@mutouzdl We are going to have some interesting merge issues when Feature branch gets merged. I am referring to my PR #1504 & for example [this line](https://github.com/ant-design-blazor/ant-design-blazor/blob/8942b61af093cdf0c3b4f83251cecc223916f490/components/form/Form.razor.cs#L161). 
Since we are here this is a good moment to discuss this future merge issue.
In that PR `EditContext` is exposed to the consumer. There the event is attached only to marshal it back to EventCallback. Here though it does much more (very interesting I must add). Can we actually do both here? It seems to me we probably shouldn't? I am not sure to be honest... I think that we should have a detection here - if EventCallback has a delegate, invoke it and skip the body of this method. Otherwise, run it. My reasoning behind it is - if a dev wants to use EventCallback, that means he/she wants to have full control over the component's behaviour. Then we should not force default behaviour on him/her.

Reply:
@anddrzejb 
I'll try to rearrange my code based on the Feature branch.

I'm going to resubmit the PR to the Feature branch.
If there are any new questions, let's discuss them in the new PR.

So I'm going to close this.